.travis.yml

```diff
@@ -57,8 +57,10 @@ script: ./scripts/runTestsOnTravis.sh $TEST_SUITE
 # Install Syft which is used by GoReleaser in the deploy step.
 # Cosign is an optional dependency to verify the Syft binary.
 before_deploy:
-  - curl -o /usr/local/bin/cosign -L https://github.com/sigstore/cosign/releases/latest/download/cosign-linux-amd64 && chmod +x /usr/local/bin/cosign
-  - curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | COSIGN_BINARY=/usr/local/bin/cosign sh -s -- -v -b /usr/local/bin v1.18.1
+  - curl -O -L "https://github.com/sigstore/cosign/releases/latest/download/cosign-linux-amd64"
```

Comment:
Given that the output before was:
```
0.33s $ curl -o /usr/local/bin/cosign -L https://github.com/sigstore/cosign/releases/latest/download/cosign-linux-amd64 && chmod +x /usr/local/bin/cosign
curl: (23) Failed writing body (0 != 1369)
The command "curl -o /usr/local/bin/cosign -L https://github.com/sigstore/cosign/releases/latest/download/cosign-linux-amd64 && chmod +x /usr/local/bin/cosign" failed and exited with 23 during .
```
... then it's possible that `curl -o` was fine but potentially that `/usr/local/bin` doesn't exist.

Has this been tested with a repository fork?